### ISSUE TITLE
DrawingFunction is nerver called on iOS

### ISSUE THREAD

**Issue by P3PPP** [chrfalch/NControl#47]
On iOS, DrawingFunction is nerver called when NControlView is added to the Layout that is set IsVisible property to false.
On Android, it works correctry.

Reproduction code:

```
public App()
{
    var nControlView = new NControlView {
        HeightRequest = 100,
        WidthRequest = 100,
        DrawingFunction = (canvas, rect) => {
            // It's never called on iOS.
            canvas.FillEllipse(rect, NGraphics.Colors.Green);
        },
    };

    var grid = new Grid {
        IsVisible = false,
        Children = {
            nControlView,
        },
    };

    var button = new Button {
        Text = "Toggle Visibility"
    };
    button.Clicked += (sender, e) => grid.IsVisible = !grid.IsVisible;

    MainPage = new ContentPage {
        Content = new StackLayout {
            VerticalOptions = LayoutOptions.Center,
            HorizontalOptions = LayoutOptions.Center,
            Children = {
                grid,
                button,
            }
        }
    };
}
```

I tapped the button and set grid.IsVisible to true, but DrawingFunction was not called.

**Comment by chrfalch** [comment]
Have you tried the last version (0.8)? It contains a call to Invalidate the
control on iOS when IsVisible is set.

**Comment by P3PPP** [comment]
Yes, I tried it on version 0.8.
Ok, I will change my code to what calling invalidate when the parent layout's IsVisble updated.
Thank you for anser.

**Comment by chrfalch** [comment]
great!

**Comment by durandt** [comment]
I've just stumbled upon this issue when upgrading from Xamarin Forms 1.5.1 to 2.3.1

NControl version was 0.7.1, I updated to 0.9.0.1 but the issue remained. 

The issue happens for `NControlView`'s rendering, whose parent is a `Grid` in another "root" `NControlView`, if I play with the root `NControlView.IsVisible`. The `DrawingFunction` of the child `NControlView` is not called when `IsVisible` is set to `true` on the root `NControlView`. 
Manually calling `Invalidate()` fixes the problem. 
Issue does not affect Android. It does seem to only affect iOS.
